renovate.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":disableRateLimiting",
+    ":maintainLockFilesWeekly",
```

Comment:
Update the pnpm lock file. This addresses the situation where non-direct dependencies may be dropped from updates.